lib/danger/commands/runner.rb

```diff
@@ -54,6 +54,19 @@ def run
       dm.print_results
     end
 
+    def self.report_error(exception)
+      case exception
+      when Interrupt
+        puts '[!] Cancelled'.red
+        Config.instance.verbose? ? raise : exit(1)
+      when SystemExit
+        raise
+      else
+        # puts UI::ErrorReport.report(exception)
```

Comment:
Yeah, it was to add support for gh-issues-inspector, it should get removed and that can be a separate pr